theme/base-2016/partials/_footer.twig

```diff
@@ -17,6 +17,8 @@
 
 </footer>
 
-<script src="{{ asset('js/jquery.min.js', 'theme') }}"></script>
+{# Extensions and copypasta gleaned from other sources tend to cause multiple jQuery
+   includes. We only include it here conditionally, if not done so already. #}
+<script>window.jQuery || document.write("<script src='{{ asset('js/jquery.min.js', 'theme') }}'><\/script>");</script>
```

Comment:
Why not just inject a DOM node instead of writing HTML? that way we follow best practices and don't inspire others to write it "the wrong way".